README.md

```diff
@@ -29,69 +54,191 @@ let extensionHandler = {
 };
 
 let fdkClient = setupFdk({
-  api_key: "<API_KEY>",
-  api_secret: "<API_SECRET>",
-  base_url: baseUrl, // this is optional
-  scopes: ["company/products"], // this is optional
-  callbacks: extensionHandler,
-  storage: new RedisStorage(redis),
-  access_mode: "offline",
+  api_key: "<API_KEY>", // API Key of an extension for authentication.
+  api_secret: "<API_SECRET>", // API Secret of an extension for authentication.
+  base_url: baseUrl, // optional. Base URL for extension.
+  scopes: ["company/products"], // optional. An array of scopes indicating the specific permissions needed for an extension.
+  callbacks: extensionHandler, // The callback function to handle extension-related tasks.
+  storage: new RedisStorage(redis), // An instance of storage (e.g., RedisStorage) for data storage.
+  access_mode: "offline", // Access mode of an extension. It can be `online` or `offline`.
+  cluster: "https://api.fyndx0.de", // optional. The API url of the Fynd Platform cluster.
+});
+
+let router = app.express.Router();
+const handlers = fdkClient.routerHandlers; // Functions that constains implementaion of OAuth
+
+router.get("/fp/install", async (req, res, next) => {
+    const { redirectUrl, fdkSession } = await handlers.fpInstall(
```

Comment:
Can handlers have better naming convention instead of `fpInstall` to `extInstall` or something more generic @jinalviranii